pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -623,7 +649,16 @@ def changed(self):
 
     @property
     def cdn_used_as_installation_source(self):
-        return self._cdnButton.get_active()
+
```

Comment:
Maybe this empty line shouldn't be here for consistency.

Reply:
Fixed. :)